apps/style/curriculum/documentation_tables.scss

```diff
@@ -0,0 +1,21 @@
+@import "color.scss";
+@import "font";
+@import "markdown";
+
+.docs-table {
```

Comment:
nit: I wonder if `docs-pages` would be a better name for this class? It's not a big deal but we could add other styling here at some point that isn't table-related. Thoughts?

Reply:
Good point!  I made this change.